orchagent/aclorch.cpp

```diff
@@ -1613,6 +1639,24 @@ bool AclOrch::processAclTableType(string type, acl_table_type_t &table_type)
     return true;
 }
 
+bool AclOrch::processAclTableStage(string stage, acl_stage_type_t &acl_stage)
+{
+    SWSS_LOG_ENTER();
+
+    auto tt = aclStageLookUp.find(toUpper(stage));
```

Comment:
>tt [](start = 9, length = 2)

tt stands for 'table type'. You should get your own name.

Reply:
good catch. Updated.